libs/vkd3d/device.c

```diff
@@ -637,6 +638,50 @@ VkInstance vkd3d_instance_get_vk_instance(struct vkd3d_instance *instance)
     return instance->vk_instance;
 }
 
+static uint32_t vkd3d_physical_device_get_time_domains(struct d3d12_device *device)
+{
+    const struct vkd3d_vk_instance_procs *vk_procs = &device->vkd3d_instance->vk_procs;
+    VkPhysicalDevice physical_device = device->vk_physical_device;
+    uint32_t i, domain_count = 0;
+    VkTimeDomainEXT *domains;
+    uint32_t result = 0;
+    VkResult vr;
+
+    if ((vr = VK_CALL(vkGetPhysicalDeviceCalibrateableTimeDomainsEXT(physical_device, &domain_count, NULL))) < 0)
+    {
+        ERR("Failed to enumerate time domains, vr %d.\n", vr);
+        return false;
```

Comment:
false?

Reply:
fixed, this used to return bool at some point.